fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1789,6 +1789,14 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = FeatureFlags.composeTopSites,
     )
 
+    /**
+     * Indicates if the shopping experience feature is enabled.
+     */
+    var enableShoppingExperience by booleanPreference(
```

Comment:
Can this be a val? Seems `SharedPreferenceUpdater` just updates the underlying preference and not this field so this is a read only property